### DISCUSSION TITLE
An official discord server

### DISCUSSION THREAD

**Discussion by Mikefizzy** [voicepaw/so-vits-svc-fork#312] [Ideas]
It would be great if we had a discord server where we share our projects and checkpoints.

Training these takes a ton of time and it would be great if we all just worked together 
instead of training the same models over and over again, 

We can also share ideas and experiments and have a great time.

**Comment by 34j** [comment]
I would like checkpoints to be submitted to HuggingFace due to security issues and compatibility with upstream repositories.

**Comment by Mikefizzy** [comment]
Obviously the files would be hosted elsewhere not discord

but the ability for people to communicate and share is crucial

**Comment by 34j** [comment]
I don't have the energy to set up a server, but someone else can do it on their own.

**Comment by Mikefizzy** [comment]
https://discord.gg/aS9zTPMvrv

here I just created a server

can you display it on the github page?

**Comment by 34j** [comment]
The server will soon collapse if you don't set up authentication, etc. By that I mean that the setup is very cumbersome.

**Comment by Mikefizzy** [comment]
alright, Ill let you know when I get that done
